src/components/SlidePanel/_slide-panel.scss

```diff
@@ -0,0 +1,15 @@
+.slide-panel {
+  background-color: white;
+  box-shadow: 0 1px 5px 1px rgba(51, 51, 51, 0.2);
+  height: calc(100% - 25px);
+  position: absolute;
+  right: 0;
+  top: 24px; // height of the top bar
```

Comment:
This is flaky because every time we update the height of the top bar this will have to be changed... maybe there is a better approach to this so that it works dynamically across the application?